impeller/entity/contents/atlas_contents.h

```diff
@@ -0,0 +1,60 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#pragma once
+
+#include <functional>
+#include <memory>
+#include <vector>
+
+#include "flutter/fml/macros.h"
+#include "impeller/entity/contents/contents.h"
+#include "impeller/entity/entity.h"
+#include "impeller/renderer/sampler_descriptor.h"
+
+namespace impeller {
+
+class AtlasContents final : public Contents {
+ public:
+  explicit AtlasContents();
+
+  ~AtlasContents() override;
+
+  void SetTexture(std::shared_ptr<Texture> texture);
+
+  std::shared_ptr<Texture> GetTexture() const;
+
+  void SetXForm(std::vector<Matrix> xform);
```

Comment:
Nit: Here and everywhere else: Maybe call this `SetTransforms`? We've been using "Transform" for transformation matrices instead of Skia's "XForm" around Impeller.

Reply:
Done